packs/feats/scholarly-defense.json

```diff
@@ -29,22 +29,22 @@
             "title": "Pathfinder #164: Hands of the Devil"
         },
         "rules": [
-          {
-            "definition": [
-              "damage:category:physical",
-              "melee",
-              {
-                "or": [
-                  "origin:trait:aberration",
-                  "origin:trait:ooze"
-                ]
-              }
-            ],
-            "key": "Resistance",
-            "type": "custom",
-            "label": "{item|name}",
-            "value": "max(1,floor(@actor.level/3))"
-          }
+            {
+                "definition": [
+                    "damage:category:physical",
+                    "melee",
+                    {
+                        "or": [
+                            "origin:trait:aberration",
+                            "origin:trait:ooze"
+                        ]
+                    }
+                ],
+                "key": "Resistance",
+                "label": "{item|name}",
+                "type": "custom",
+                "value": "max(1,floor(@actor.level/3))"
+            }
```

Comment:
?

Reply:
> ?

Sorry, I don't know how that got there. I didn't touch that file as far as I could tell.